medusa/classes.py

```diff
@@ -129,15 +130,17 @@ def __init__(self, item, download_current_quality):
         self.actual_episodes = None
 
     def add_result_to_cache(self, cache):
-        # Cache the item if needed
+        """Cache the item if needed."""
         if self.add_cache_entry:
             logger.debug('Adding item from search to cache: {release_name}', release_name=self.name)
```

Comment:
No this works perfectly. @ratoaq2 added it. I've been using it in numerous of locations.